function_template.jinja

```diff
@@ -38,13 +39,18 @@ impl ScalarUDFImpl for Func {
         &self.signature
     }
 
+    // start implementing return_type
     fn return_type(&self, _arg_types: &[DataType]) -> Result<DataType> {
-        Ok(DataType::Boolean)
+        todo!()
     }
+    // end implementing return_type
 
-    fn invoke(&self, args: &[ColumnarValue]) -> Result<ColumnarValue> {
-        // start function impl
+    // start implementing invoke
+    fn invoke(&self, _args: &[ColumnarValue]) -> Result<ColumnarValue> {
         todo!()
```

Comment:
Default `invoke` body should not be a panic -- it should return a `DatafusionError::NotImplemented`, with a nice error message (preferrably containing a link to this file telling the user where to fill the implementation).